tests/executors/test_celery_executor.py

```diff
@@ -75,6 +75,17 @@ def _prepare_app(broker_url=None, execute=None):
     patch_app = mock.patch('airflow.executors.celery_executor.app', test_app)
     patch_execute = mock.patch('airflow.executors.celery_executor.execute_command', test_execute)
 
+    backend = test_app.backend
```

Comment:
Curious, in the SQLite tests this is failing with `KeyError: backend`

@auvipy @mik-laj Any idea how it's possible for the app to _not_ have a backend from here? https://github.com/apache/airflow/blob/5c3248cc4c57be34d868482cf1034fb1d6bfd0bf/tests/executors/test_celery_executor.py#L179-L189

Reply:
Ah okay, I know what's gone on. 

Looking closer at the logs in CI I see 

> E   ModuleNotFoundError: No module named '# overridden by startup scripts'

Previously we weren't ever actually creating a backend, so it didn't matter. I think the fix here is to limit this test to mysql+postgres backends only

Sorry for the ping @auvipy for what was (clearly) going to be something we've done.

Reply:
Fixed by https://github.com/apache/airflow/pull/8909/files#diff-6342f2098d57976d779e20623d3ff4d5R179-R181

Happy with that change @mik-laj?

Reply:
It looks good. SQLite is not a good backend for testing distributed applications.